appinfo/info.xml

```diff
@@ -19,6 +19,10 @@
 	<licence>agpl</licence>
 	<author homepage="https://georg.coffee">Georg Ehrke</author>
 	<author homepage="https://tcit.fr">Thomas Citharel</author>
+	<author homepage="christoph.wurst@nextcloud.com">Christoph Wurst</author>
```

Comment:
drop the email addresses. those are optional and will only cause us spam :)

Reply:
lol, those aren't even supposed to be email addresses. Time to go home